### ISSUE TITLE
Rename and restructure classes and packages

### ISSUE THREAD

**Issue by bonaparten** [GIScience/ohsome-api#125]
A general review of all classes' and packages' names and structures would be useful.

For example, `AggregateRequestExecutor.java` and `DataRequestExecutor.java` could be renamed to `AggregationRequestExecutor.java` and `ExtractionRequestExecutor.java`. So these classes will follow the same naming criteria used in other packages, i.e. in the package `output`: `DefaultAggregationResponse.java ` and `ExtractionResponse.java `.

**Comment by FabiKo117** [comment]
I agree, it would definitely make sense to review again the current structures and names. After we've merged https://github.com/GIScience/ohsome-api/pull/124 and https://github.com/GIScience/ohsome-api/pull/121 we can think about which other classes and packages would need to be renamed/restructured.

**Comment by bonaparten** [comment]
I pushed [this old commit](https://github.com/GIScience/ohsome-api/commit/a9e1f50625b8d7e7d2902afff12620b22c934665). It is an example of how we can restructure some packages and classes.

**Comment by Hagellach37** [comment]
We are moving towards ohsome API v2.0.
Hence, this topic will not be followed up on.

Feel free to raise it again if it is still relevant for the new API version.
